32-explore_pathway_separation.Rmd

```diff
@@ -0,0 +1,416 @@
+---
+title: "Pathway 'separation'"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**J. Taroni 2018**
+
+We've noted that pathways or genesets that are similar to one another, such as 
+cell types of the myeloid lineage (e.g., neutrophils and monocytes/macrophages),
+can tend to get "lumped together" in PLIER models trained on training sets with
+smaller sample sizes or less relevant biological contexts.
+In contrast, the MultiPLIER model (~37K samples) tends to "separate" similar 
+pathways into different latent variables.
+(See `06-sle-wb_cell_type` and `07-sle_cell_type_recount2_model`.)
+
+**Here, we're interested in the notion of "pathway separation" when different 
```

Comment:
can also use the term "disentangled features" ;)

Reply:
True! I think "pathway separation" is probably a little bit more clinician- and biologist-friendly.